gradle/compile-native-multiplatform.gradle

```diff
@@ -1,10 +1,17 @@
+import kotlin.KotlinVersion
+
 /*
  * Copyright 2016-2021 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
  */
 
 project.ext.nativeMainSets = []
 project.ext.nativeTestSets = []
 
+// TODO: this block should be removed when Kotlin version is updated to 1.9.20
+// Right now it is used for conditional removal of native targets that will be removed in 1.9.20 (iosArm32, watchosX86)
+// and is necessary for testing of Kotlin dev builds.
+def enableDeprecatedNativeTargets = !KotlinVersion.CURRENT.isAtLeast(1, 9, 20)
```

Comment:
doesn't this test the version of gradle's embedded kotlin and not the kotlin version with which coroutines are built?

Reply:
Mm, Im not sure about that: 
KotlinVersion.CURRENT `Returns the current version of the Kotlin standard library.`

Reply:
It's easy to check -- update Kotlin version in our build and see if this property changes its value

Reply:
Seems to be the way indeed; sorry for the confusion